demos/twoway/cyclow/es6/src/phonebooth.js

```diff
@@ -0,0 +1,26 @@
+import {Block} from 'cyclow'
+
+const Phonebooth = () => Block({
+  on: {
+    'in.init': () => state => ({name: 'Steve'}),
+    'dom.text': text => state => ({name: text}),
+    'dom.click': () => ({
+      state: () => ({name: 'Batman'}),
+      'dom.action': el => el.firstElementChild.focus()
```

Comment:
Neat! And I had never seen `firstElementChild` before, learning about that was an extra bonus for me :)